web/src/toga_web/deploy/config.toml

```
@@ -0,0 +1,2 @@
+pyscript_version = "2024.11.1"
+backend = "pyscript"
```

From a future-proofing point of view, it might be advisable to break up this definition so that pyscript-specific configuration is flagged as such:
```suggestion
backend = "pyscript"

[pyscript]
version = "2024.11.1"
```